doc/dev/GETTING_STARTED.md

```diff
@@ -81,59 +81,89 @@ Note: The following steps were tested with Visual Studio Community 2022 using Wi
 - Download and install [cmake][cmake-download]
 - Download and install [Visual Studio 2022 Community Edition][visual-studio]
 
-#### Recover F3D sources
+#### Retrieve VTK and F3D sources
 
 - Open git bash
 
 ```sh
 cd C:
 mkdir dev
 cd dev
+mkdir vtk
+cd vtk
+git clone https://gitlab.kitware.com/vtk/vtk.git src
```

Comment:
```suggestion
git clone --depth 1 --branch v9.4.1 https://gitlab.kitware.com/vtk/vtk.git src
```
and remove the three following lines

Reply:
Does not work, you cant clone a tag.

Reply:
It's supposed to work according to https://stackoverflow.com/a/21699307

Reply:
damn it does.